src/interfaces.ts

```diff
@@ -153,6 +153,16 @@ export interface PrecomputedFlag {
   doLog: boolean;
 }
 
+export interface IPrecomputedBandit {
+  action: string | null;
```

Comment:
major: action should never be null
```suggestion
  action: string;
```

Reply:
done